docs/src/main/sphinx/admin/fault-tolerant-execution.md

```diff
@@ -550,6 +550,10 @@ load:
 exchange.base-directories=s3://exchange-spooling-bucket-1,s3://exchange-spooling-bucket-2
 ```
 
+While the exchange manager is designed to support S3-compatible storage systems,
```

Comment:
Would be better to move it into the intro sentence in line 531 I think .. then this is all together.